src/test/java/ContactListEqualsTest.java

```diff
@@ -5,51 +5,42 @@ public class ContactListEqualsTest {
     @Test
     @SuppressWarnings("UnstableApiUsage")
     public void equals_verify_contract() {
-        ContactList emptyA = new ContactList();
-        ContactList emptyB = new ContactList();
-        ContactList emptyC = new ContactList(100);
+        ContactList emptyA = of();
+        ContactList emptyB = of();
+        ContactList emptyC = ofCapacity();
 
-        ContactList singletonA = new ContactList();
-        ContactList singletonB = new ContactList();
-        ContactList singletonC = new ContactList(100);
-        singletonA.add(new Friend("Bob", "Smith", "bsmith@gmail.com"));
-        singletonB.add(new Friend("Bob", "Smith", "bsmith@gmail.com"));
-        singletonC.add(new Friend("Bob", "Smith", "bsmith@gmail.com"));
+        ContactList singletonA = of(new Friend("Bob", "Smith", "bsmith@gmail.com"));
```

Comment:
Constants or other functions for `bob` and others would also help clean this up. I'd probably do functions to avoid having `bob() == bob()`

Reply:
ack, but _meh_